src/app/modules/loose-components.module.ts

```diff
@@ -221,6 +222,7 @@ import { OrganizationBadgeModule } from "./vault/modules/organization-badge/orga
     MasterPasswordPolicyComponent,
     NavbarComponent,
     NestedCheckboxComponent,
+    OrganizationSwitcherComponent,
```

Comment:
I think it's best to avoid putting new components in here.
Can this possibly be put in any of the new modules created with other EUVR work? (maybe the `organization-vault` or `vault` modules?)

Reply:
Really we need a new `OrganizationsModule` to house all these org components. The `vault` modules are too low level - the org switcher sits outside the vault component and is visible when navigating the other org pages as well. I think this is a refactor we can do after merging all of this, it feels too late in the game to go doing it now. (Ideally all of these "loose components" should be broken out into their own feature modules.)